src/main/java/com/xiaomi/infra/pegasus/client/request/BatchDelete.java

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2020, Xiaomi, Inc.  All rights reserved.
+// This source code is licensed under the Apache License Version 2.0, which
+// can be found in the LICENSE file in the root directory of this source tree.
+package com.xiaomi.infra.pegasus.client.request;
+
+import com.xiaomi.infra.pegasus.client.PegasusTableInterface;
+import io.netty.util.concurrent.Future;
+import java.util.List;
+
+public class BatchDelete extends Batch<Delete, Void> {
```

Comment:
```suggestion
public class BatchDelete extends Batch<Delete, Void> {
```

I think DeleteBatch is perferred. Or BatchedDelete.